src/Testcontainers.LocalStack/LocalStackBuilder.cs

```diff
@@ -0,0 +1,134 @@
+            .WithEnvironment("EXTERNAL_SERVICE_PORTS_END", port);
+    }
+    
+    /// <summary>
+    /// Sets the LocalStack Services.
+    /// </summary>
+    /// <param name="services">The LocalStack services.</param>
+    /// <returns>A configured instance of <see cref="LocalStackBuilder" />.</returns>
+    public LocalStackBuilder WithServices(params AwsService[] services)
+    {
+        if (services?.Length is null)
+        {
+            return this;
+        }
+
+        string servicesNames = string.Join(",", services.Select(service => service.Name));
+        return Merge(DockerResourceConfiguration, new LocalStackConfiguration(services: services))
+            .WithEnvironment("SERVICES", servicesNames);
+    }
+
+    
+    /// <inheritdoc />
+    public override LocalStackContainer Build()
+    {
+        Validate();
+        return new LocalStackContainer(DockerResourceConfiguration, TestcontainersSettings.Logger);
+    }
+
+    protected override LocalStackBuilder Init()
+    {
+        return base.Init()
+            .WithImage(LocalStackImage)
+            .WithExternalServicePortStart("4510")
+            .WithExternalServicePortEnd("4559")
+            .WithServices()
+            .WithPortBinding(LocalStackPort, true)
+            .WithWaitStrategy(Wait.ForUnixContainer().AddCustomWaitStrategy(new UntilReady()));
+    }
+
+    /// <inheritdoc />
+    protected override void Validate()
+    {
+        base.Validate();
+
+        _ = Guard.Argument(DockerResourceConfiguration.ExternalServicePortStart, nameof(DockerResourceConfiguration.ExternalServicePortStart))
+            .NotNull()
+            .NotEmpty();
+        
+        _ = Guard.Argument(DockerResourceConfiguration.ExternalServicePortEnd, nameof(DockerResourceConfiguration.ExternalServicePortEnd))
+            .NotNull()
+            .NotEmpty();
+    }
+
+    /// <inheritdoc />
+    protected override LocalStackBuilder Clone(IContainerConfiguration resourceConfiguration)
+    {
+        return Merge(DockerResourceConfiguration, new LocalStackConfiguration(resourceConfiguration));
+    }
+
+    /// <inheritdoc />
+    protected override LocalStackBuilder Clone(IResourceConfiguration<CreateContainerParameters> resourceConfiguration)
+    {
+        return Merge(DockerResourceConfiguration, new LocalStackConfiguration(resourceConfiguration));
+    }
+
+    /// <inheritdoc />
+    protected override LocalStackBuilder Merge(LocalStackConfiguration oldValue, LocalStackConfiguration newValue)
+    {
+        return new LocalStackBuilder(new LocalStackConfiguration(oldValue, newValue));
+    }
+    
+    private sealed class UntilReady : IWaitUntil
+    {
+        public async Task<bool> UntilAsync(IContainer container)
+        {
+            var (stdout, _) = await container.GetLogs()
+                .ConfigureAwait(false);
+            return stdout != null && stdout.Contains("Ready.\n");
+        }
+    }
```

Comment:
```suggestion
    /// <inheritdoc cref="IWaitUntil" />
    private sealed class UntilReady : IWaitUntil
    {
        /// <inheritdoc />
        public async Task<bool> UntilAsync(IContainer container)
        {
            var (stdout, _) = await container.GetLogs()
                .ConfigureAwait(false);
            return stdout != null && stdout.Contains("Ready.\n");
        }
    }
```

Reply:
I just removed whole class and used better wait until method